Dockerfile

```diff
@@ -9,6 +9,7 @@ COPY . /logflare
 
 
 WORKDIR /logflare
+ENV CARGO_NET_GIT_FETCH_WITH_CLI=true
```

Comment:
@filipecabaco could you also add in an inline comment here to say why we need this in the image, so that we don't accidentally remove it when refactoring